src/System.Windows.Forms/src/System/Windows/Forms/LinkLabel.Link.LinkLabelLinkAccessibleObject.cs

```diff
@@ -0,0 +1,189 @@
+
+                public override AccessibleRole Role
+                {
+                    get
+                    {
+                        return AccessibleRole.Link;
+                    }
+                }
+
+                public override AccessibleStates State
+                {
+                    get
+                    {
+                        AccessibleStates state = AccessibleStates.Focusable;
+
+                        // Selected state
+                        //
+                        if (_ownerLink.Owner.FocusLink == _ownerLink)
+                        {
+                            state |= AccessibleStates.Focused;
+                        }
+
+                        return state;
+                    }
+                }
+
+                public override string Value
+                {
+                    get
+                    {
+                        // Narrator announces Link's text twice, once as a Name property and once as a Value, thus removing value.
+                        // Value is optional for this role (Link).
+                        return string.Empty;
+                    }
+                }
+
+                public override void DoDefaultAction()
+                {
+                    _ownerLink.Owner.OnLinkClicked(new LinkLabelLinkClickedEventArgs(_ownerLink));
+                }
+
+                internal override bool IsIAccessibleExSupported() => true;
+
+                internal override object? GetPropertyValue(UiaCore.UIA propertyID)
+                {
+                    if (propertyID == UiaCore.UIA.IsEnabledPropertyId)
+                    {
+                        if (!_ownerLink.Owner.Enabled)
+                        {
+                            return false;
+                        }
+                    }
+
+                    return base.GetPropertyValue(propertyID);
+                }
+
+                private int _nextIndex => _linkIndex + 1;
+
+                private int _prevIndex => _linkIndex - 1;
+
+                internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
+                    => direction switch
+                    {
+                        UiaCore.NavigateDirection.Parent => _linkLabelAccessibleObject,
+                        UiaCore.NavigateDirection.NextSibling
+                            => -1 < _nextIndex && _nextIndex < _linkLabelAccessibleObject.GetChildCount()
+                                ? _linkLabelAccessibleObject.GetChild(_nextIndex)
+                                : null,
+                        UiaCore.NavigateDirection.PreviousSibling
+                            => -1 < _prevIndex && _prevIndex < _linkLabelAccessibleObject.GetChildCount()
+                                ? _linkLabelAccessibleObject.GetChild(_nextIndex)
+                                : null,
+                        _ => base.FragmentNavigate(direction),
+                    };
```

Comment:
No, you need to make it like this:

```suggestion
                internal override UiaCore.IRawElementProviderFragment? FragmentNavigate(UiaCore.NavigateDirection direction)
                    => direction switch
                    {
                        UiaCore.NavigateDirection.Parent => _linkLabelAccessibleObject,
                        UiaCore.NavigateDirection.NextSibling
                            => _linkLabelAccessibleObject.GetChild(CurrentIndex + 1),
                        UiaCore.NavigateDirection.PreviousSibling
                            => _linkLabelAccessibleObject.GetChild(CurrentIndex - 1),
                        _ => base.FragmentNavigate(direction),
                    };
```